consume-more-stuff/server/server.js

```diff
@@ -47,14 +49,32 @@ app.use(bodyParser.urlencoded({ extended: true }));
 //     });
 //   })
 
-app.use("/users", usersRouter);
-app.use("/items", itemsRouter);
-app.use("/categories", categoriesRouter);
+app.use("/api/users", usersRouter);
+app.use("/api/items", itemsRouter);
+app.use("/api/categories", categoriesRouter);
 
-app.get("/", (req, res) => {
-  console.log("smoke test");
-  res.send("smoke test");
-});
+app.post(`/api/login/:username`, (req, res) => {
```

Comment:
Should have caught this earlier but we should rely on the username to come across in the `req.body` rather than `req.params`.  

So the route s/b `.post('/api/login')` and the username s/b pulled in from the form. When encrypting a form this would keep usernames hidden over the wire and not in the URL which can be more easily detected.